client/src/main/java/io/pravega/client/tables/TableConfiguration.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.tables;
+
+
+import io.pravega.client.stream.ScalingPolicy;
+import java.io.Serializable;
+import lombok.Builder;
+import lombok.Data;
+
+/**
+ * The configuration of a Table.
+ */
+@Data
+@Builder
+public class TableConfiguration implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * The Scope of the Table.
+     *
+     */
+    private final String scope;
+
+    /**
+     * The name of the Table.
+     *
+     */
+    private final String tableName;
+
+    /**
+     * The Table's Scaling Policy..
+     *
+     */
+    private final ScalingPolicy scalingPolicy;
+
+    public static final class TableConfigurationBuilder {
+        private ScalingPolicy scalingPolicy = ScalingPolicy.fixed(1);
```

Comment:
You are talking about an implementation detail. I simply defined contracts here. Whatever stub code is in here is subject to be replaced in subsequent PRs.

Reply:
More or less. You are right that I'm referring to implementation, but what I'm really thinking is that perhaps we should think of evolving the interface rather than introduce calls that we know we won't serve in the first version. This is to avoid the wrong expectation on what features are available. Will it be difficult to add this later?

Reply:
I don't know how difficult it will be to do it later. Now I'm just adding contracts. 

I have removed this field.